Binding/src/main/scala/com/thoughtworks/binding/Binding.scala

```diff
@@ -327,7 +328,7 @@ object Binding extends MonadicFactory.WithTypeClass[Monad, Binding] {
       * @note This method must not be invoked inside a `@dom` method body or a `Binding { ... }` block.
       */
     def value_=(newValue: A): Unit = {
-      if (cache != newValue) {
+      if (cache.isInstanceOf[Mutable] || cache.isInstanceOf[TraversableView[_, _]] ||cache != newValue) {
```

Comment:
`||cache` there is missing space after `||` :wink: 
The same below :point_down: